functions/model/tasks/tasks.js

```diff
@@ -26,6 +27,10 @@ exports.tasks = functions.https.onRequest((request, response) => {
             return logWork(request, response);
         } else if (mode == "delete") {
             return deleteTask(request, response);
+        } else if (mode == "getTasks") {
+            return getTasks(request, response);
+        } else if (mode == "getTasksCard") {
+            return getTasksCard(request, response);
```

Comment:
based on above changes we can make these calls to simple method and that will do everything